### ISSUE TITLE
Test and validation set

### ISSUE THREAD

**Issue by BakerBunker** [redmist328/APNet2#2]
Hi, the paper mentioned "We randomly selected 12,000 audio clips for training, 100 for validation, and 500 for testing.", could you provide the filename list of these three sets?

**Comment by redmist328** [comment]
I divided the dataset according to the method used in [VITS](https://github.com/jaywalnut310/vits/tree/main/filelists), and I apologize for the miswriting in the preprint version. I also appreciate your pointing it out.   :P

**Comment by redmist328** [comment]
Hello @BakerBunker . I divided the dataset according to the method used in [VITS](https://github.com/jaywalnut310/vits/tree/main/filelists), and I apologize for the miswriting in the preprint version. I also appreciate your pointing it out. :P
